yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -52,17 +53,17 @@ export class Archiver implements L2BlockSource, UnverifiedDataSource, ContractDa
    * @param publicClient - A client for interacting with the Ethereum node.
    * @param rollupAddress - Ethereum address of the rollup contract.
    * @param inboxAddress - Ethereum address of the inbox contract.
-   * @param unverifiedDataEmitterAddress - Ethereum address of the unverifiedDataEmitter contract.
+   * @param contractDeploymentEmitterAddress - Ethereum address of the contractDeploymentEmitter contract.
```

Comment:
Once again, would remove this param since it never will be a separate contract from now on.

Reply:
contract deployment events are still needed right? `UnverifiedDataEmitter` already had this functionality, now it's the only thing it will do, hence `contractDeploymentEmitter`